packages/tracing/src/hubextensions.ts

```diff
@@ -64,15 +52,29 @@ function sample<T extends Transaction>(hub: Hub, transaction: T, samplingContext
 
   // if the user has forced a sampling decision by passing a `sampled` value in their transaction context, go with that
   if (transaction.sampled !== undefined) {
+    transaction.tags = { ...transaction.tags, samplingMethod: 'explicitly_set' };
```

Comment:
The known/valid sampling methods could be defined in a central location to 1) avoid typos; and 2) document what the possible values are.

Reply:
That's a good idea. I'll leave this convo unresolved until I make the change.